Turn 'deployment considerations' into 'deployment path' (#3456)
theforeman/foreman-documentation: guides/common/modules/con_additional-deployment-tasks.adoc

Reviewer: mental note: I should create a follow-up PR to add Puppet and Salt as optional enhancements; maybe even ACD and KernelCare. :shrug:

[id="additional-deployment-tasks_{context}"]
= Additional deployment tasks

{Project} offers a range of additional capabilities that you can use to further enhance your {Project} deployment.

> For example:

Author: Without "For example", it might look like this is supposed to be a complete list. Which it isn't, it's just an overview of the additional tasks we want to highlight. So no change for now, please feel free to explain further if I'm missing something :)

Remote execution commands on hosts::
With remote execution, you can perform various tasks on multiple hosts simultaneously.
{Project} supports the following modes of transport for remote execution: pull-based mode (over MQTT/HTTPS) and push-based mode (over SSH).
+

> For more information, see link:{ManagingHostsDocURL}Configuring_and_Setting_Up_Remote_Jobs_managing-hosts[Configuring and setting up remote jobs] in _{ManagingHostsDocTitle}_.

Reviewer: ```suggestion
For more information, see {ManagingHostsDocURL}Configuring_and_Setting_Up_Remote_Jobs_managing-hosts[Configuring and setting up remote jobs] in _{ManagingHostsDocTitle}_.
```

`link:` in combination with `{XDocURL}` breaks things for me in downstream.

Automating tasks with Ansible::
ifndef::satellite[]
You can enable the Ansible plugin on your {ProjectServer}.
endif::[]
With Ansible, you can automate repetitive tasks on {Project} hosts.
+

> For more information, see link:{ManagingConfigurationsAnsibleDocURL}[_{ManagingConfigurationsAnsibleDocTitle}_].

Reviewer: ```suggestion
For more information, see {ManagingConfigurationsAnsibleDocURL}[_{ManagingConfigurationsAnsibleDocTitle}_].
```

Security management with OpenSCAP::
ifndef::satellite[]
You can enable the OpenSCAP plugin on your {ProjectServer} and any {SmartProxyServers}.
endif::[]
With OpenSCAP, you can manage compliance policies and run security compliance scans on your hosts.
After the scan completes, a compliance report is uploaded to your {ProjectServer}.
+

> For more information, see link:{ManagingSecurityDocURL}[_{ManagingSecurityDocTitle}_].

Reviewer: ```suggestion
For more information, see {ManagingSecurityDocURL}[_{ManagingSecurityDocTitle}_].
```

Load balancing::
With load balancing configured on your {SmartProxyServers}, you can improve performance on {SmartProxyServers} while also improving performance and stability for host connections to {Project}.
+

> For more information, see link:{ConfiguringLoadBalancerDocURL}[_{ConfiguringLoadBalancerDocTitle}_].

Reviewer: ```suggestion
For more information, see {ConfiguringLoadBalancerDocURL}[_{ConfiguringLoadBalancerDocTitle}_].
```

> ifdef::satellite[]

Reviewer: ```suggestion
ifdef::satellite,orcharhino[]
```

Incident management with Red{nbsp}Hat Insights::
With Red{nbsp}Hat Insights enabled on your {ProjectServer}, you can identify key risks to stability, security, and performance.
+

> For more information, see link:{InstallingServerDocURL}[Using Red{nbsp}Hat Insights with Satellite Server] in _{InstallingServerDocTitle}_.

Reviewer: ```suggestion
For more information, see {InstallingServerDocURL}[Using Red{nbsp}Hat Insights with Satellite Server] in _{InstallingServerDocTitle}_.
```

+
ifndef::foreman-deb,satellite[]
For more information on using Salt with {Project}, see {ManagingConfigurationsSaltDocURL}[_{ManagingConfigurationsSaltDocTitle}_].
You will need to enable the Salt plugin on your {ProjectServer}.
endif::[]

Security management with OpenSCAP::
ifndef::satellite[]
You can enable the OpenSCAP plugin on your {ProjectServer} and any {SmartProxyServers}.
endif::[]
With OpenSCAP, you can manage compliance policies and run security compliance scans on your hosts.
After the scan completes, a compliance report is uploaded to your {ProjectServer}.
+
For more information, see {ManagingSecurityDocURL}[_{ManagingSecurityDocTitle}_].

Load balancing::
With load balancing configured on your {SmartProxyServers}, you can improve performance on {SmartProxyServers} while also improving performance and stability for host connections to {Project}.
+
For more information, see {ConfiguringLoadBalancerDocURL}[_{ConfiguringLoadBalancerDocTitle}_].

ifdef::satellite,orcharhino[]
Incident management with Red{nbsp}Hat Insights::
With Red{nbsp}Hat Insights enabled on your {ProjectServer}, you can identify key risks to stability, security, and performance.
+

> For more information, see {InstallingServerDocURL}[Using Red{nbsp}Hat Insights with Satellite Server] in _{InstallingServerDocTitle}_.

Reviewer: ```suggestion
For more information, see {InstallingServerDocURL}[Using Red{nbsp}Hat Insights with {ProjectServer}] in _{InstallingServerDocTitle}_.
```
